include/podio/utilities/TypeHelpers.h

```diff
@@ -50,35 +50,29 @@ namespace det {
 
 namespace detail {
 
-  /**
-   * Helper struct to determine whether a given type T is in a tuple of types
-   * that act as a type list in this case
-   */
+  /// Helper struct to determine whether a given type T is in a tuple of types
+  /// that act as a type list in this case
   template <typename T, typename>
   struct TypeInTupleHelper : std::false_type {};
 
   template <typename T, typename... Ts>
   struct TypeInTupleHelper<T, std::tuple<Ts...>> : std::bool_constant<(std::is_same_v<T, Ts> || ...)> {};
 
-  /**
-   * variable template for determining whether type T is in a tuple with types
-   * Ts
-   */
+  /// variable template for determining whether type T is in a tuple with types
+  /// Ts
   template <typename T, typename Tuple>
   static constexpr bool isInTuple = TypeInTupleHelper<T, Tuple>::value;
 
-  /**
-   * Helper struct to turn a tuple of types into a tuple of a template of types, e.g.
-   *
-   * std::tuple<int, float> -> std::tuple<std::vector<int>, std::vector<float>>
-   * if the passed template is std::vector
-   *
-   * NOTE: making the template template parameter to Template variadic because
-   * clang will not be satisfied otherwise if we use it with, e.g. std::vector.
-   * This will also make root dictionary generation fail. GCC works without this
-   * small workaround and is standard compliant in this case, whereas clang is
-   * not.
-   */
+  /// Helper struct to turn a tuple of types into a tuple of a template of types, e.g.
+  ///
+  /// std::tuple<int, float> -> std::tuple<std::vector<int>, std::vector<float>>
+  /// if the passed template is std::vector
+  ///
+  /// NOTE: making the template template parameter to Template variadic because
```

Comment:
Should we use [`\note`](https://www.doxygen.nl/manual/commands.html#cmdnote) in these cases?

Reply:
Yes, and done.